website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 
+
+- [`spec.requirements`](../reference/provisioner-crd#specrequirements)
+  - Cloud Provider Agnostic 
+  - Kubernetes Well Known Labels
+  - This section includues generally applicable constraints (zone, instance type) that each cloud provider is expected to implement. 

```

Comment:
"includes" is misspelled. Are constraints such as zone, instance type, etc required or optional? Can we can get clarity from engineering?